### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by Abdullahoda** [IBAX-io/go-ibax#778] [Bug Submission]
When I log in with the phone in Weaver, the texts overlap and do not fit on the screen.   It needs to be renewed, it is difficult to create an ecosystem from here
![Screenshot_20220125_215708_com android chrome](https://user-images.githubusercontent.com/97207588/151041929-11b418f2-327c-436d-866d-7450b4e28dc8.jpg)
![Screenshot_20220125_215633_com android chrome](https://user-images.githubusercontent.com/97207588/151041957-d987d705-c284-4465-ba08-d45025fc34d3.jpg)

**Comment by ibaxio** [comment]
Weaver is an IDE tool for developers and we are cooperate with JUTKEY team.

Please refer to this discussion: #629
